graphdatascience/tests/integration/test_training_pipeline.py

```diff
@@ -19,12 +19,28 @@ def G(runner: Neo4jQueryRunner, gds: GraphDataScience) -> Generator[Graph, None,
         (a: Node),
         (b: Node),
         (c: Node),
+        (d: Node),
+        (e: Node),
+        (f: Node),
+        (g: Node),
         (a)-[:REL]->(b),
         (a)-[:REL]->(c),
+        (a)-[:REL]->(d),
+        (a)-[:REL]->(e),
+        (a)-[:REL]->(f),
+        (a)-[:REL]->(g),
         (b)-[:REL]->(c),
         (b)-[:REL]->(a),
+        (b)-[:REL]->(d),
+        (b)-[:REL]->(e),
+        (b)-[:REL]->(f),
+        (b)-[:REL]->(g),
         (c)-[:REL]->(a),
-        (c)-[:REL]->(b)
+        (c)-[:REL]->(b),
+        (c)-[:REL]->(d),
+        (c)-[:REL]->(e),
+        (c)-[:REL]->(f),
+        (c)-[:REL]->(g)
```

Comment:
NIT: you could have also changed the splitConfig

Reply:
I tried experimenting with that, but couldn't get it to work. Always some of the datasets not big enough. This solution seems better to me